idr0053-study.txt

```diff
@@ -0,0 +1,93 @@
+# STUDY DESCRIPTION SECTION				
+				
+Comment[IDR Study Accession]	idr0053			
+Study Title	Virtual nanoscopy: Generation of ultra-large high resolution electron microscopy maps			
+Study Type	high resolution electron microscopy			
+Study Type Term Source REF				
+Study Type Term Accession				
+Study Description	A key obstacle in uncovering the orchestration between molecular and cellular events is the vastly different length scales on which they occur. We describe here a methodology for ultrastructurally mapping regions of cells and tissue as large as 1 mm(2) at nanometer resolution. Our approach employs standard transmission electron microscopy, rapid automated data collection, and stitching to create large virtual slides. It greatly facilitates correlative light-electron microscopy studies to relate structure and function and provides a genuine representation of ultrastructural events. The method is scalable as illustrated by slides up to 281 gigapixels in size. Here, we applied virtual nanoscopy in a correlative light-electron microscopy study to address the role of the endothelial glycocalyx in protein leakage over the glomerular filtration barrier, in an immunogold labeling study of internalization of oncolytic reovirus in human dendritic cells, in a cryo-electron microscopy study of intact vitrified mouse embryonic cells, and in an ultrastructural mapping of a complete zebrafish embryo slice.		 	
+Study Organism	Danio rerio, Mus musculus, Homo sapiens			
+Study Organism Term Source REF	NCBITaxon			
+Study Organism Term Accession	7955, 10090, 9606			
+Study Experiments Number	1			
+Study External URL				
+Study Public Release Date				
+				
+# Study Publication				
+Study PubMed ID	22869601			
+Study Publication Title	Virtual nanoscopy: Generation of ultra-large high resolution electron microscopy maps			
+Study Author List	Frank G.A. Faas, M. Cristina Avramut, Bernard M. van den Berg, A. Mieke Mommaas, Abraham J. Koster, Raimond B.G. Ravelli			
+Study PMC ID	PMC3413355 			
+Study DOI	https://dx.doi.org/10.1083/jcb.201201140			
+				
+# Study Contacts				
+Study Person Last Name	Faas			
+Study Person First Name	Frank			
+Study Person Email				
+Study Person Address				
+Study Person ORCID				
+Study Person Roles				
+				
+# Study License and Data DOI				
+Study License	CC-BY 4.0			
```

Comment:
Previous IDR datasets already coming from JCB (idr0004) were published with `CC-BY-NC-SA`. Might be something we want to clarify